globset/src/lib.rs

```diff
@@ -449,7 +449,7 @@ impl GlobSet {
 
 /// GlobSetBuilder builds a group of patterns that can be used to
 /// simultaneously match a file path.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
```

Comment:
This is a change to the API. It seems fine in principle. But could you please remove the `derive(Default)` and write out an explicit `Default` impl that calls `GlobSetBuilder::new()`? This way, we make sure the implementations are exactly the same.